From cca909511ead5023f4a9f71412d79411a384326d Mon Sep 17 00:00:00 2001
From: Andrey Zhizhikin <andrey.zhizhikin@leica-geosystems.com>
Date: Mon, 13 Jan 2020 17:32:19 +0000
Subject: [PATCH] devregs: add support for imx8mn soc

i.MX8M Nano SoC is very close to i.MX8M Mini, therefore we would allow
them to use the same register description file.

Upstream-Status: Submitted [https://github.com/boundarydevices/devregs/pull/5]

Signed-off-by: Andrey Zhizhikin <andrey.zhizhikin@leica-geosystems.com>
---
 src/devregs.cpp | 4 ++++
 1 file changed, 4 insertions(+)

diff --git a/src/devregs.cpp b/src/devregs.cpp
index 67c52eb..d540894 100644
--- a/src/devregs.cpp
+++ b/src/devregs.cpp
@@ -717,6 +717,10 @@ static int getcpu(unsigned &cpu, const char *path) {
 				cpu = 0x82;
 				break;
 			}
+			if (strstr(inBuf, "i.MX8MN")) {
+				cpu = 0x82;
+				break;
+			}
 			if (!get_rev(inBuf, "Revision", &cpu))
 				if (cpu != 0x10)
 					break;
-- 
2.17.1

